spec/controllers/tags_controller_spec.rb

```diff
@@ -22,8 +22,37 @@
     context "successful creation" do
       before { post :create, params: { tag: tag_params } }
 
+      it "creates a new, non-canonical, tag" do
+        expect(flash[:notice]).to eq "Tag was successfully created."
```

Comment:
You can use the `it_redirects_to_with_notice` helper which also checks the destination path.